numpy/distutils/fcompiler/gnu.py

```diff
@@ -244,6 +250,26 @@ class Gnu95FCompiler(GnuFCompiler):
     compiler_aliases = ('gfortran',)
     description = 'GNU Fortran 95 compiler'
 
+    _global_libs = set()
+    _local_libs = {}
+    _setup_py_dirs = []
+
+    def __init__(self, *args, **kw):
+        GnuFCompiler.__init__(self, *args, **kw)
+        try:
+            with open("runtime_shared_libs.txt") as fp:
```

Comment:
The only required global lib is actually the OpenBLAS dll, so this should be moved to system_info in the future. However, this is good enough for now.